### ISSUE TITLE
What error should be returned when trying to access position from insecure origin?

### ISSUE THREAD

**Issue by yellowdoge** [w3c/geolocation-api#12]
Chromium issue [702244](https://bugs.chromium.org/p/chromium/issues/detail?id=702244)  says Chromium returns a `PERMISSION_DENIED`, which sounds legit, but Safari might be returning a `POSITION_UNAVAILABLE`.  I think Chromium response is OK and the alleged Safari response might be incorrect, but just in case it'd be great to clarify what's the expected error code when accessing from an insecure origin.

**Comment by miguelao** [comment]
+@jyasskin

**Comment by mkruisselbrink** [comment]
FWIW the [example](https://w3c.github.io/webappsec-secure-contexts/#legacy-example) in the Secure Contexts spec says to return `PERMISSION_DENIED`, but of course that's not normative.

**Comment by miguelao** [comment]
I think the WebKit line where the error is produced is https://github.com/WebKit/webkit/blob/master/Source/WebCore/Modules/geolocation/Geolocation.cpp#L366 :
```cpp
    if (shouldBlockGeolocationRequests()) {
        notifier->setFatalError(PositionError::create(PositionError::POSITION_UNAVAILABLE, ASCIILiteral(originCannotRequestGeolocationErrorMessage)));
        return;
    }
```

**Comment by reillyeon** [comment]
This was resolved by #34.
